objax/functional/loss.py

```diff
@@ -21,11 +21,13 @@
            'sigmoid_cross_entropy_logits']
 
 from typing import Union, Iterable, Optional
-
 import jax.numpy as jn
-
 from objax.functional.core import logsumexp
 from objax.typing import JaxArray
+import warnings
+
+
+warn_mess = 'Argument shape mismatch in'
```

Comment:
The main reason I avoided to use the function names explicitly was to keep the code length lesser than 120 characters per line. Otherwise the linter returns an error. Now I have added an another constant `FUNC_NAME` in each functions definition and then passed it to the warning message. I guess this one should be perfect. Kindly do verify.

Reply:
Yes, that's fine.
FYI, if line is too long you can split it into multiple lines.